fault-proof/src/proposer.rs

```diff
@@ -39,6 +39,7 @@ where
     P: Provider<Ethereum> + Clone + Send + Sync,
 {
     pub config: ProposerConfig,
+    pub l1_address: Address,
```

Comment:
Is this the `prover_address`? If so, let's call it that and then add an environment variable for configuring.